src/foundation/y_code_pal_pseudo_comments.dtel.xml

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_DTEL" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <DD04V>
+    <ROLLNAME>Y_CODE_PAL_PSEUDO_COMMENTS</ROLLNAME>
+    <DDLANGUAGE>E</DDLANGUAGE>
+    <DOMNAME>BOOLE</DOMNAME>
+    <HEADLEN>55</HEADLEN>
+    <SCRLEN1>10</SCRLEN1>
+    <SCRLEN2>20</SCRLEN2>
+    <SCRLEN3>40</SCRLEN3>
+    <DDTEXT>Code Pal - Ignore Pseudo Comments</DDTEXT>
+    <REPTEXT>PseudCom</REPTEXT>
+    <SCRTEXT_S>PseudCom</SCRTEXT_S>
+    <SCRTEXT_M>PseudCom</SCRTEXT_M>
+    <SCRTEXT_L>PseudCom</SCRTEXT_L>
```

Comment:
Do we use this field on the screen? If so, it would be good to have the description properly. 

DDTEXT = Code Pal - Ignore Pseudo Comments
REPTEXT = Ignore Pseudo Comments
SCRTEXT_L = Ignore Pseudo Comments
SCRTEXT_M = Ignore Pseudo Comments
SCRTEXT_S = Ign Ps Com